README.md

```diff
@@ -13,22 +13,41 @@ a unified interface describing heterogeneous build jobs.
 Hosted [here](https://awslabs.github.io/aws-build-accumulator/).
 
 
-### Requirements
+### Installation
 
-* Python 3
+You can install the AWS Build Accumulator by using [Homebrew](https://brew.sh):
+
+```bash
+brew tap aws/tap
+brew install litani
+```
+
+Running `man` will inform you of all capabilities inside this package. Run:
+
+```bash
+man litani
+```
+
+### Dependencies
+
+If you are cloning the source code, you will need the following dependencies:
+
+#### Requirements
```

Comment:
Since you're now phrasing this in terms of "Dependencies"  `Requirements` -> `Required`